tests/e2e/cucumber/steps/ui/spaces.ts

```diff
@@ -154,13 +154,15 @@ When(
 )
 
 When(
-  '{string} navigates to the trashbin of the project space {string}',
+  /^"([^"]*)" navigates to the trashbin(| of the project space "([^"]*)")$/,
```

Comment:
does this step work like this?
`"Alice" navigates to the trashbin of the project space "Personal"`

Reply:
No, it would not work. If there is not any project space created, then by default trashbin will navigate to Personal space trashbin.
In case, there is a project space then only there will be options inside the trashbin to choose between Personal space or the project space.

Reply:
Ahh! :+1: